### PR TITLE
Migrating WT3 screen model

### PR DESCRIPTION
**Pull request by luohezhiming**
## Summary/Motivation:

Add screen model

## Changes proposed in this PR:
- add screen unit model originally developed by @MarkusHolly
- add costing  model
- add corresponding tests

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: watertap/unit_models/zero_order/tests/test_screen_zo.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                value(model.fs.unit.properties_treated[0].conc_mass_comp["foo"]))
+        assert (pytest.approx(0.001, rel=1e-5) ==
+                value(model.fs.unit.properties_byproduct[0].flow_vol))
+        assert (pytest.approx(300.0, rel=1e-5) ==
+                value(model.fs.unit.properties_byproduct[0].conc_mass_comp["bod"]))
+        assert (pytest.approx(600.0, rel=1e-5) ==
+                value(model.fs.unit.properties_byproduct[0].conc_mass_comp["tss"]))
+        assert (pytest.approx(1e-5, rel=1e-5) ==
+                value(model.fs.unit.properties_byproduct[0].conc_mass_comp["foo"]))
+        assert (pytest.approx(1.002*0*3600, abs=1e-5) ==
```
Comment (kurbansitterley) [inline watertap/unit_models/zero_order/tests/test_screen_zo.py]: Don't we want to have an actual value here?

Comment (kurbansitterley) [inline watertap/unit_models/zero_order/tests/test_screen_zo.py]:
```suggestion
        assert (pytest.approx(0, abs=1e-5) ==
```

Does the test pass with 0?

**Review by kurbansitterley — CHANGES_REQUESTED** [review]
Should probably be testing the value that the model returns rather than an expression (even if that value is zero).

File: watertap/unit_models/zero_order/tests/test_screen_zo.py (hunk 1 above)
Comment (luohezhiming) [reply watertap/unit_models/zero_order/tests/test_screen_zo.py]: Revised to 0

Comment (luohezhiming) [inline watertap/unit_models/zero_order/tests/test_screen_zo.py]:
```suggestion
        assert (pytest.approx(0, abs=1e-5) ==
```

**Review by kurbansitterley — APPROVED** [review]
LGTM

**Review by andrewlee94 — APPROVED** [review]

**Pull request merged**